zhmccli/_cmd_partition.py

```diff
@@ -916,6 +967,133 @@ def cmd_partition_console(cmd_ctx, cpc_name, partition_name, options):
             "{exc}: {msg}".format(exc=exc.__class__.__name__, msg=exc))
 
 
+def cmd_partition_dump(cmd_ctx, cpc_name, partition_name, **options):
+    # pylint: disable=missing-function-docstring
+
+    client = zhmcclient.Client(cmd_ctx.session)
+    partition = find_partition(cmd_ctx, client, cpc_name, partition_name)
+
+    name_map = {
+        # These parameters are the same for CPCs with and without the storage
+        # management feature. Feature-specific parameters are added further
+        # down:
+        'configuration': 'dump-configuration-selector',
+        'parameters': 'dump-os-specific-parameters',
+        'lba': 'dump-record-lba',
+
+        # Options that are handled in this function:
+        'volume': None,
+        'timeout': None,
+    }
+    options = original_options(options)
+    properties = options_to_properties(options, name_map)
+    volume_opt = options['volume']  # It is required
+    timeout = options['timeout']  # It is optional but has a default
+
+    if storage_management_feature(partition):
+
+        # Check and parse the --volume option into SG.SV or UUID
+        volume_parts = volume_opt.split('.')
```

Comment:
This doesn't work since `.` can be used in a volume name and storage name.
The allowed pattern for volume and storage name is `[a-zA-Z0-9_.@- ]{1,64}` (please double check).

We can either use a separate cmdline option or, for example, `/` as delimiter.

Reply:
Good point. I have double checked the names for SG and SV and the pattern you show is correct.
DONE - changed to using `/` as a delimiter, so it is now `SG/SV`.

For consistency, I also changed the delimiter used for the string without storage mgmt feature from dot to slash, so it is now `HBA/WWPN/LUN`.